src/data/users.tsx

```diff
@@ -263,6 +263,15 @@ const Users: User[] = [
     source: "https://github.com/ParnaRoyChowdhury777/Age-Calculator",
     tags: ["opensource", "product", "react", "css", "javascript"],
   },
+  {
+    title: "To Do List App",
+    description:
+      "A simple To-Do List App using HTML CSS JS . Add your tasks and manage them easily. ",
+    preview: require("./showcase/todolist.png"),
+    website: "https://50-days-50-web-project.vercel.app/TO_DO_LIST/todolist.html",
```

Comment:
@dhairyagothi can you update you web page UI, layout  for best practices and good page for motivate and good for users to work build like this and for best UI/UX